src/Cache/QueryCache.php

```diff
@@ -16,14 +16,12 @@
  */
 final class QueryCache
 {
-    private CacheInterface $cache;
     private bool $enabled = true;
     public array $info = [];
     private ?int $duration = 3600;
 
-    public function __construct(CacheInterface $cache)
+    public function __construct(private CacheInterface $cache)
```

Comment:
The package supports PHP version '7.4':
https://github.com/yiisoft/db/blob/master/composer.json#L22

Reply:
hi, this is a development branch, and we agree to support `PHP 8.0` at a minimum.